src/app/routes/enrolment/view-requests/components/expiration-info/expiration-info.component.html

```diff
@@ -1,11 +1,12 @@
 <app-report-problem>
   <ng-container *ngIf="defaultValidityPeriod">
-    Default expiration date of this role is <b>{{ defaultValidityPeriod | timeShift | defaultDate }} UTC</b>.
+    The default expiration date for this role is <b>{{ defaultValidityPeriod | timeShift | defaultDate }} UTC</b>.
   </ng-container>
   <ng-container *ngIf="!defaultValidityPeriod"> Default expiration date is not set. </ng-container>
-  This role will be issued with
+
   <ng-container *ngIf="expirationTime"
-    ><b>{{ expirationTime | timeShift | defaultDate }} UTC</b></ng-container
+    >This role will now be issued with an expiration date of
+    <b>{{ expirationTime | timeShift | defaultDate }} UTC</b></ng-container
```

Comment:
Does this line make sense to you @dawidgil @whitneypurdum ? Is this the use of the Bitwise OR https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Bitwise_OR ?

Reply:
Yeah, I wasn't sure about it either but it is something with maybe Angular syntax? It converts timestamps to formatted date strings. I'm not sure where it comes from, it took me a moment to realize that it was transforming the timestamps.

Reply:
Those are so called pipes: https://angular.io/guide/pipes. They are transforming data in html. It's easier to reuse and test them instead of writing the same code in component class.

Reply:
@dawidgil Thanks for the link!